source/overview/implementation_plan.rst

```diff
@@ -0,0 +1,369 @@
+Implementation Plan
+================================
+
+1   Introduction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1.1 Purpose
+.........................................
+
+Mattermost is an open source, private cloud alternative to proprietary SaaS messaging solutions designed to offer organizations superior control, data sovereignty, configurability, freedom from lock-in, and enhanced security by keeping vital communications behind your firewall. You have chosen to implement Mattermost in our supported `editions:`_
+
+.. _`editions:`: https://about.mattermost.com/features/
+
+ - Team Edition
+
+ - Enterprise Edition (E10) which adds additional features to the platform including AD/LDAP integration, multi-factor authentication ("MFA"), custom branding, advanced access control policy and next business day support.
+
+ - Enterprise Edition (E20) includes E10 features plus security controls, compliance reporting, high availability, and enterprise class support. 
+
+The purpose of this Implementation Plan is to:
+
+ - Describe the solution to be deployed;
```

Comment:
At first it was hard to make an association that `Describe the solution to be deployed` was section 1.2, `Outline the process for implementing the solution (key personnel, tasks, schedule, etc.) was section 2, etc.

One option might to re-number 1.2 --> 2, 2 --> 3, 3 --> 4 to make it more clear, or consider matching these bullet points with the section headings to reinforce the connection.

I'm 0/5 though.

Reply:
Updated